rollup.config.js

```diff
@@ -46,6 +49,12 @@ const rollup = {
 			targets: ['distribution/assets'],
 			runOnce: true, // `false` would be nice, but it deletes the files too early, causing two extension reloads
 		}),
+		svelte({
+			compilerOptions: {
+				customElement: true,
```

Comment:
Svelte is probably the best solution here:

- no JSX conflicts
- dedicated .svelte file that can live next to the .tsx file as `<feature-name>.svelte`
- custom elements support, so these components can be used in JSX as `<rgh-something count={1}/>`